ReactiveUsersMicroservice/src/main/java/Cloud2024/ReactiveUsersMicroservice/Presentation/UserController.java

```diff
@@ -29,6 +28,35 @@ public Mono<UserBoundary> createUser (
                 .log();
     }
 
+    @GetMapping(
+            path = "/criteria",
+            produces = {MediaType.TEXT_EVENT_STREAM_VALUE})
+    public Flux<UserBoundary> getUsersByCriteria(
```

Comment:
It's looks good, I am not sure the the GetAllUsers that what the led to the url conflicts.
I suggest you the comment all the end point and enable them one by one to find which one is making the problem